build.gradle

```diff
@@ -25,20 +25,18 @@ import org.ajoberstar.gradle.git.tasks.*
 buildscript {
     repositories {
         mavenCentral()
-        maven {
-            url 'http://dl.bintray.com/jfrog/jfrog-jars'
-        }
+        jcenter()
```

Comment:
Oh handy. I just noticed it replacing the old plain URL with the latest artifactory plugin, didn't test deeper :-)